Solution1/ConsoleApp/Constants.Validation.User.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ConsoleApp
+{
+    public static partial class Constants
+    {
+        public static partial class Validation
```

Comment:
refactor your constants: make to classes: Validation (for user validation), Errors(for user errors),
LoggerMessages (for all logger messages),  LoggerMessages is not obligatory, usually logger messages are written without constants

Reply:
+